app/models/concerns/aggregatable.rb

```diff
@@ -67,6 +67,15 @@ class AggregateListError < StandardError; end
   def add_item_from_child_list(item)
     raise AggregateListError.new('add_item_from_child_list method only available on aggregate lists') unless aggregate_list?
 
+    if item.unit_weight
+      other_items = child_lists.map(&:list_items)
+      other_items.flatten!
```

Comment:
I wanted to do this in one line but rubocop-performance suggests that way is slower.